nss-day-cohort-48/giffygram-jumpstation
# Favorite posts

#### Changes Made

1. added FavoritePosts.js to feed directory.
1. Modified PostList.js to include the user name of who made a post.
1. modified main.js. created a new variable currentUser that pulls the user id from local storage and send to api
when a new post is created. 

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout favorite-posts

2. Open a new Terminal tab (⌘T) and navigate to the server directory.

3. to test create a new post, expected outcome is seeing the poster name under the post on giffyfeed
